codespell_lib/data/dictionary.txt

```diff
@@ -20077,9 +20109,16 @@ rouine->routine
 rouines->routines
 round-robbin->round-robin
 roundign->rounding
+roundtriped->roundtripped, round-tripped, round tripped,
+roundtriping->roundtripping, round-tripping, round tripping,
+roundtripp->roundtrip
```

Comment:
Can we add the other suggestions to this one too please?

Reply:
specifically?

Reply:
`roundtrip, round-trip, round trip,`

Reply:
Done, 2909386